crates/turborepo-lib/src/commands/daemon.rs

```diff
@@ -0,0 +1,62 @@
+use std::path::PathBuf;
+
+use super::CommandBase;
+use crate::{
+    cli::DaemonCommand,
+    daemon::{DaemonConnector, DaemonError},
+};
+
+/// Runs the daemon command.
+pub async fn main(command: &DaemonCommand, base: &CommandBase) -> Result<(), DaemonError> {
+    let (dont_start, dont_kill) = match command {
+        DaemonCommand::Status { .. } => (true, true),
+        DaemonCommand::Restart | DaemonCommand::Stop => (true, false),
+        DaemonCommand::Start => (false, false),
+    };
```

Comment:
Is this idiomatic Rust? (I'm not the person to ask.) It feels very strange to have the assignment be the driver of behavior.

Reply:
I didn't want the DaemonConnector to be aware of the Command, and this looks to me like quite a concise and declarative way to set the flags.